server/lib/parser/transaction.js

```diff
@@ -16,59 +16,20 @@ function parse(entry, txs) {
     counter++;
 
     if (counter % socketThrottle === 0) {
+
+      counter = 0;
+
       io.sockets.emit('tx', {
         txid: txJSON.hash,
-        valueOut: tx.outputs.reduce((a, b) => {
-          a = a.toJSON();
-          b = b.toJSON();
+        valueOut: tx.outputs.reduce((sum, tx) => {
+          tx = tx.toJSON();
 
-          const valA = (a.value || a.valueOut.value || 0) / 1e8;
-          const valB = (b.value || b.valueOut.value || 0) / 1e8;
+          const valB = (tx.value || tx.valueOut.value || 0) / 1e8;
 
-          return valA + valB;
-        }),
-      }, 0);
+          return sum + valB;
+        }, 0),
+      });
     }
-
-    const t = new TxModel({
```

Comment:
Transactions are being saved. They are saved as subdocuments on the Block model. Separate parsers were only needed for relational models for populate.